peachpy/x86_64/function.py

```diff
@@ -690,6 +692,18 @@ def propogate_avx_backward(block, instructions, avx_state):
                     virtual_register.virtual_id, conflict_internal_ids)
             output_registers = instruction.output_registers
 
+            # Notify all live virtual registers about conflicts with any live
+            # physical registers.
+            live = Register._reconstruct_multiple(instruction._live_registers)
+            virtuals = set(l for l in live if l.is_virtual)
+            physicals = live - virtuals
+            physical_conflicts = set(physical._internal_id
```

Comment:
This is not right. Physical registers are not in conflict with a virtual register, if they are of different kind (e.g. `XMMRegister` and `rax`) or if they have non-overlapping masks (e.g. `ah` and `GeneralPurposeRegister8`)